src/main/java/appeng/client/gui/widgets/GuiCraftingCPUTable.java

```diff
@@ -161,17 +162,26 @@ public void drawFG(int offsetX, int offsetY, int mouseX, int mouseY, int guiLeft
                     GL11.glTranslatef(x + CPU_TABLE_SLOT_WIDTH - 19, y + 3, 0);
                     parent.drawItem(0, 0, craftingStack.getItemStack());
                 } else {
-                    final int iconIndex = 16 * 4 + 3;
-                    parent.bindTexture("guis/states.png");
-                    final int uv_y = iconIndex / 16;
-                    final int uv_x = iconIndex - uv_y * 16;
 
                     GL11.glScalef(0.5f, 0.5f, 1.0f);
                     GL11.glColor4f(1.0F, 1.0F, 1.0F, 1.0F);
-                    parent.drawTexturedModalRect(0, 0, uv_x * 16, uv_y * 16, 16, 16);
-                    GL11.glTranslatef(18.0f, 2.0f, 0.0f);
-                    GL11.glScalef(1.5f, 1.5f, 1.0f);
-                    font.drawString(cpu.formatStorage(), 0, 0, GuiColors.CraftingStatusCPUStorage.getColor());
+                    parent.bindTexture("guis/states.png");
+                    parent.drawTexturedModalRect(0, 0, 3 * 16, 4 * 16, 16, 16);
+                    GL11.glColor4f(0.5F, 0.5F, 0.5F, 1.0F);
+                    parent.drawItem(
+                            16 * 4,
+                            0,
+                            AEApi.instance().definitions().blocks().craftingAccelerator().maybeStack(1).orNull());
+                    GL11.glColor4f(1.0F, 1.0F, 1.0F, 1.0F);
+                    GL11.glTranslatef(18.0f, 6.0f, 0.0f);
+                    GL11.glScalef(1.1f, 1.1f, 1f);
+
+                    font.drawString(cpu.formatStorageWithoutB(), 0, 0, GuiColors.CraftingStatusCPUStorage.getColor());
```

Comment:
![image](https://github.com/user-attachments/assets/d1b39503-db2a-40aa-b7e5-2abb300d8d11)
with 'B' is a little bit of crowding, but not a big deal.

Reply:
Then you can move icon and count of count of co-processors little bit to the right. Or make this part of GUI wider (this will require texture modification). But I'm sure that units should be displayed to prevent confusion.

Reply:
![image](https://github.com/user-attachments/assets/fac981bf-365c-45f3-baaf-669a16a78adc)